interfaces/udev/udev_test.go

```diff
@@ -189,7 +189,6 @@ func (s *uDevSuite) TestReloadUDevRulesRunsUDevAdmWithTwoSubsystems(c *C) {
 		{"udevadm", "control", "--reload-rules"},
 		{"udevadm", "trigger", "--subsystem-nomatch=input"},
 		{"udevadm", "trigger", "--subsystem-match=input"},
-		{"udevadm", "trigger", "--subsystem-match=tty"},
```

Comment:
So, just to check I understand this right: tty is dropped because of `--subsystem-nomatch=input` right?

Reply:
correct!